components/add_user_to_group_multiselect/add_user_to_group_multiselect.tsx

```diff
@@ -80,6 +82,8 @@ export default class AddUserToGroupMultiSelect extends React.PureComponent<Props
             values: [],
             term: '',
             loadingUsers: true,
+            maxValues: undefined,
```

Comment:
Just curious why these variables are put in state rather than just calculating them in the `render()` method?

Reply:
Yeah agreed, moved to the render function